upgrade-notes/upgrade_notes.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -x
```

Comment:
I would remove the `x` and add the `e` modifier
so script will stop on error and executed commands are not printed

Reply:
Got it.